vdev/src/testing/runner.rs

```diff
@@ -305,6 +325,10 @@ impl ContainerTestRunner for DockerTestRunner {
     fn image_name(&self) -> String {
         env::var("ENVIRONMENT_UPSTREAM").unwrap_or_else(|_| UPSTREAM_IMAGE.to_string())
     }
+
+    fn needs_docker_sock(&self) -> bool {
+        false
+    }
```

Comment:
This strikes me as odd, since it's returning a static value?

Reply:
It is static there, but the other implementation of the trait uses a value from the struct, so it's not static everywhere.